modules/dynamic-plugins/proc-install-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,129 @@
+[id="proc-install-third-party-plugins-rhdh_{context}"]
+= Installing third-party plugins in {product}
+
+You can install a third-party plugins in {product} without rebuilding the {product-very-short} application.
+
+The location of the `dynamic-plugin-config.yaml` file depends on the deployment method. For more details, refer to xref:proc-config-dynamic-plugins-rhdh-operator_rhdh-installing-rhdh-plugins[] and xref:con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[].
+
+Plugins are defined in the `plugins` array within the `dynamic-plugin-config.yaml` file. Each plugin is represented as an object with the following properties:
+
+* `package`: The plugin's package definition, which can be an OCI image, a TGZ file, a JavaScript package, or a directory path
+* `disabled`: A boolean value indicating whether the plugin is enabled or disabled.
+* `integrity`: The integrity hash of the package, required for TGZ file and JavaScript packages.
+* `pluginConfig`: The plugin's configuration. For backend plugins, this is optional; for frontend plugins, it is required. The `pluginConfig` is a fragment of the `app-config.yaml` file, and any added properties are merged with the {product-very-short} `app-config.yaml` file.
+
+.Prerequisites
+* The third-party plugin is packaged as a dynamic plugin in one of the following formats:
+** OCI image
+** TGZ file
+** JavaScript package
+
+For more information about packaging a third-party plugin, see xref:proc-package-third-party-dynamic-plugin_{context}[].
+
+[NOTE]
+====
+You can also load dynamic plugins from a plain directory, though this is intended for development or testing purposes and is not recommended for production, except for plugins included in the {product-very-short} container image.
+====
+
+.Procedure
+Load a plugin packaged as an OCI image::
++
+--
+. Define the plugin with the `oci://` prefix in the following format in `dynamic-plugins.yaml` file:
++
+`oci://<image-name>:<tag>!<plugin-name>`
++
+.Example configuration in `dynamic-plugins.yaml` file
+[source,yaml]
+----
+plugins:
+  - disabled: false
+    package: oci://quay.io/example/image:v0.0.1!backstage-plugin-myplugin
+----
+
+. Configure authentication for private registries by setting the `REGISTRY_AUTH_FILE` environment variable to the path of the registry configuration file. For example, `~/.config/containers/auth.json` or `~/.docker/config.json`.
+
+. To perform an integrity check, use the image digest in place of the tag in `dynamic-plugins.yaml` file as follows:
```

Comment:
"...in place of the tag in the 'dynamic-plugins.yaml' file..."

I think we should add an article before 'dynamic-plugins.yaml' as we are using the identifier "file" after that.

Reply:
+1, added the article.